backend/LfClassicData/LfClassicMiniLcmApi.cs

```diff
@@ -230,28 +246,38 @@ private async IAsyncEnumerable<Entry> Query(QueryOptions? options = null, string
             })));
     }
 
-    private static Entry ToEntry(Entities.Entry entry)
+    private async Task<Entry> ToEntry(Entities.Entry entry)
     {
+        Sense[] senses;
+        if (entry.Senses is null)
+        {
+            senses = [];
+        }
+        else
+        {
+            var senseTasks = entry.Senses.OfType<Entities.Sense>().Select(sense => ToSense(entry.Guid, sense));
```

Comment:
I'm worried about a cache stampede here. There's the potential for this to run the GetPartOfSpeech method many times in parallel since we're kicking off a bunch of tasks and not awaiting them until they've all started, this would result in a bunch of those tasks all seeing our cache as null so they call query all parts of speech. I'll just fix this here

Reply:
Got it. Since there are usually fewer than 5 senses (usually only 1 for most entries), the cost of doing this in sequence instead of parallel won't be too great, and it'll certainly be less than the cost of a cache stampede doing 5 queries at once. Thanks for catching that.